runtime/src/system_instruction_processor.rs

```diff
@@ -33,7 +33,7 @@ struct Address {
 }
 
 impl Address {
-    fn is_signer(&self, signers: &HashSet<Pubkey>) -> bool {
+    fn is_instruction_account_signer(&self, signers: &HashSet<Pubkey>) -> bool {
```

Comment:
I don't think this rename is necessary, why did you change it?

Reply:
Good spotting, was regex replace going to far.